src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -78,6 +78,34 @@
         </v-btn>
       </div>
 
+      <div
+        v-else-if="showContinueInNowButton"
+        class="d-flex justify-center my-1"
+      >
+        <v-btn
+          v-if="showOpenExternalIcon"
```

Comment:
Go to Colin if FF doesn't support continuation in for this entity. Otherwise, we'll show the "Continue In Now" button.